src/decoder/lattice-simple-decoder.cc

```diff
@@ -564,8 +565,11 @@ void LatticeSimpleDecoder::ProcessNonemitting() {
   for (unordered_map<StateId, Token*>::iterator iter = cur_toks_.begin();
        iter != cur_toks_.end();
        ++iter) {
-    queue.push_back(iter->first);
-    best_cost = std::min(best_cost, iter->second->tot_cost);
+    StateId key = iter->first;
+    if (fst_->NumInputEpsilons(key) != 0) {
```

Comment:
@vimalmanohar I'm getting an error here because of this method. Can you help with this?

Reply:
It looks like fst_ is not a pointer. In Dan's PR, it was changed to a pointer. So here you need to use `fst_.`